server/api/controller/sign.js

```diff
@@ -33,7 +33,7 @@ class SignController extends Controller {
 				// if there is a user but active, this will return an error
 
 				return this.model
-					.forge(req.body)
+					.forge({ ...req.body, status:0})
```

Comment:
@florpor am not sure about this one
the status is required but its not part of the form the front end sends

Reply:
it's assigned in the `assignValues` function of the model class [here](https://github.com/meirim-org/meirim/blob/28e3320a1680bebf95f38ec28e30e20130e387e8/server/api/model/person.js#L45).
tests run fine without this change as far as i can see

Reply:
the E2E tests? weird! they were failing for me before this change